client/v2/client.go

```diff
@@ -119,6 +123,49 @@ func NewHTTPClient(conf HTTPConfig) (Client, error) {
 	}, nil
 }
 
+// Pings cluster
+func (c *client) Ping(timeout time.Duration) (time.Duration, string, error) {
```

Comment:
I guess my comment was meant more as "put the guts of the description on the method doc, and the shorter description on the interface field" :smile: 

And yes, we always appreciate some `golint` tidying!